stock_picking_mixins/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Stock Picking Mixins",
+    "summary": """
+        Adds a structure to insert the Picking Status field for purchase and sale orders.
```

Comment:
I'd prefer a more generic summary, like this:
```suggestion
        Adds generic structures related to stock picking models
```